wake-lock/idlharness.https.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Wake Lock API IDL tests</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://w3c.github.io/wake-lock/">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/WebIDLParser.js"></script>
+<script src="/resources/idlharness.js"></script>
+<script>
+"use strict";
+
+function doTest([dom_idl, wakelock_idl, wakelock]) {
+  const idl_array = new IdlArray();
+  idl_array.add_untested_idls(dom_idl);
+  idl_array.add_untested_idls('interface Navigator {};');
+  idl_array.add_idls(wakelock_idl);
+  self.wakelock = wakelock;
+  self.request = wakelock.createRequest();
+  idl_array.add_objects({WakeLock: ['wakelock'], WakeLockRequest: ['request']});
+  idl_array.test();
+}
+
+promise_test(() => {
+  return Promise.all([
+    fetch("/interfaces/dom.idl").then((response) => response.text()),
+    fetch("/interfaces/wake-lock.idl").then((response) => response.text()),
+    navigator.getWakeLock("screen")
+  ]).then(doTest);
+}, "Test driver");
```

Comment:
Nit: This should describe the test a bit better... see above.

Reply:
This is a standalone test for `promise_test` itself, I just want to keep consistent with most of IDL tests in w-p-t. :)